scripts/ci/perf_test.sh

```diff
@@ -49,42 +30,29 @@ while [ -z $ec2_ip ]; do
       --query "Reservations[].Instances[].PublicIpAddress" \
       --output text)
 done
-
-trap "aws --region us-west-2 ec2 terminate-instances --instance-ids $ec2_instance_id" KILL
+echo "export EC2_IP=$ec2_ip" >> vars.sh
 
 # push binary and script to instance
 debname=$(find /tmp/workspace/artifacts/influxdb2*amd64.deb)
 base_debname=$(basename $debname)
 source_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
 
-scp influx_tools ubuntu@$ec2_ip:/home/ubuntu/influx_tools
+# On the first try, add the host key to the list of known hosts
+until ssh -o StrictHostKeyChecking=no ubuntu@$ec2_ip echo Connected ; do
```

Comment:
The old circleci machine executor appears to have had StrictHostKeyChecking=no in the machine definition. The new docker instance (correctly) does not, so we prime the pump here in order to save the host key.